runners/direct-java/src/test/java/org/apache/beam/runners/direct/InProcessPipelineRunnerTest.java

```diff
@@ -57,15 +71,15 @@ public void wordCountShouldSucceed() throws Throwable {
 
     PCollection<KV<String, Long>> counts =
         p.apply(Create.of("foo", "bar", "foo", "baz", "bar", "foo"))
-            .apply(MapElements.via(new SimpleFunction<String, String>() {
+            .apply(via(new SimpleFunction<String, String>() {
```

Comment:
not a huge fan of this static import - hurts readability (unlike checkNotNull ;).